pkgs/applications/networking/mailreaders/mblaze/default.nix

```diff
@@ -26,6 +29,16 @@ stdenv.mkDerivation rec {
     install -Dm644 -t $out/share/zsh/site-functions contrib/_mblaze
   '' + lib.optionalString (ruby != null) ''
     install -Dt $out/bin contrib/msuck contrib/mblow
+
+    for x in `find $out/bin "(" -type f -executable ")" -or -type l`
+    do
+      wrapProgram $x \
```

Comment:
I think we should probably only wrap the programs that are shell scripts. The others probably don’t need it, and Oulu feel weird to me to wrap programs that didn’t need it.

It would also be totally fine to decide that the others are out of scope for this PR, and to just concentrate on mcom for now, and fix the others later or not at all. Up to you — depends how quickly you want to get this over with!